rooms/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db      import models
+
+from core.models    import TimeStampModel
+
+class Category(models.Model):
+    name            = models.CharField(max_length = 50)
+    img_url         = models.CharField(max_length = 200)
+
+    class Meta:
+        db_table    = 'categories'
+
+class RoomType(models.Model):
+    name            = models.CharField(max_length = 50)        
+
+    class Meta:
+        db_table    = 'room_types' 
+
+class Room(TimeStampModel):
+    name                 = models.CharField(max_length = 50)
+    address              = models.CharField(max_length = 50)
+    detail_address       = models.CharField(max_length = 50)
+    price                = models.DecimalField(max_digits = 10, decimal_places = 2)
+    description          = models.TextField()
+    latitude             = models.CharField(max_length = 50)
+    longitude            = models.CharField(max_length = 50)
+    maximum_occupancy    = models.IntegerField()
+    bedroom              = models.IntegerField()
+    bathroom             = models.IntegerField()
+    bed                  = models.IntegerField()
+    host                 = models.ForeignKey('hosts.Host', on_delete = models.CASCADE)
+    category             = models.ForeignKey('Category', on_delete = models.CASCADE)
+    room_type            = models.ForeignKey('RoomType', on_delete = models.CASCADE)
+
+class RoomFacility(models.Model):
+    name                 = models.CharField(max_length = 50)
+    rooms_room_facility  = models.ManyToManyField(
+        'Room',
+        through          = 'RoomsRoomFacility',   
+        through_fields   = ('room_facility', 'room')  
+    ) 
+
+    class Meta:
+        db_table    = 'room_facilities' 
+
+class RoomsRoomFacility(models.Model):
+    room            = models.ForeignKey('Room', on_delete = models.CASCADE)
+    room_facility   = models.ForeignKey('RoomFacility', on_delete = models.CASCADE)
+
+    class Meta:
+        db_table    = 'rooms_room_facilities'
```

Comment:
```suggestion
    room_type            = models.ForeignKey('RoomType', on_delete = models.CASCADE)
    facilities = models.ManyToManyField('Facility', through='RoomFacility')

class Facility(models.Model):
    name                 = models.CharField(max_length = 50)

    class Meta:
        db_table    = 'facilities' 

class RoomFacility(models.Model):
    room            = models.ForeignKey('Room', on_delete = models.CASCADE)
    room_facility   = models.ForeignKey('RoomFacility', on_delete = models.CASCADE)

    class Meta:
        db_table    = 'rooms_facilities'
```

ManyToMany는 이런식으로 작성해야 할 것 같습니다.

Reply:
room에서 facilities를 접근해야하는 것이기 때문에 room쪽에 필드를 추가했습니다.
```python
room = Room.objects.get(id=1)
room.facilities ...
```

Reply:
다대다의 경우 어느쪽에 해당 필드를 추가해야하는지 어려웠는데 
들어주신 예시처럼 생각해보면 조금 더 쉽게 모델링 할 수 있을 것 같네요..! 반영했습니다!